ExampleApp/ExampleApp/Common/UpdatableCell/CellUpdatable.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+protocol UpdatableCell {
+
+    func update(with item: Any)
```

Comment:
Again, not sure about the Any type here. Maybe it's ok to provide the associated type?